scaleway/testdata/provider-instance-ip-zones.cassette.yaml

```diff
@@ -230,24 +212,21 @@ interactions:
     form: {}
     headers:
       User-Agent:
-      - scaleway-sdk-go/v1.0.0-beta.7+dev (go1.18.2; linux; amd64) terraform-provider/develop
+      - scaleway-sdk-go/v1.0.0-beta.7+dev (go1.19.5; darwin; amd64) terraform-provider/develop
         terraform/terraform-tests
-    url: https://api.scaleway.com/instance/v1/zones/fr-par-2/ips/07eae38f-a230-4586-9d58-1d1476f2d07b
+    url: https://api.scaleway.com/instance/v1/zones/fr-par-2/ips/77e0cacb-6b15-4c65-b60f-47011b210fce
     method: GET
   response:
-    body: '{"ip": {"id": "07eae38f-a230-4586-9d58-1d1476f2d07b", "address": "51.159.151.134",
-      "reverse": null, "server": null, "organization": "ee7bd9e1-9cbd-4724-b2f4-19e50f3cf38b",
-      "project": "ee7bd9e1-9cbd-4724-b2f4-19e50f3cf38b", "zone": "fr-par-2", "tags":
-      []}}'
+    body: '{"ip":{"address":"51.159.152.92","id":"77e0cacb-6b15-4c65-b60f-47011b210fce","organization":"105bdce1-64c0-48ab-899d-868455867ecf","project":"105bdce1-64c0-48ab-899d-868455867ecf","reverse":null,"server":null,"tags":[],"zone":"fr-par-2"}}'
```

Comment:
Couldn't find where this query is coming?

Reply:
It's from this: https://github.com/scaleway/terraform-provider-scaleway/blob/f5685ad6f550c0c815821408787dc0ae87f0f232/scaleway/provider_test.go#L362